roxie/ccd/ccdserver.cpp

```diff
@@ -8374,10 +8374,12 @@ class CRoxieServerSortActivity : public CRoxieServerActivity
             sortAlgorithm = (sortFlags & TAFspill) ? spillingMergeSortAlgorithm : mergeSortAlgorithm;
         else if (stricmp(algorithmName, "parmergesort")==0)
             sortAlgorithm = (sortFlags & TAFspill) ? spillingParallelMergeSortAlgorithm : parallelMergeSortAlgorithm;
+#ifdef _USE_TBB
         else if (stricmp(algorithmName, "tbbqsort")==0)
             sortAlgorithm = tbbQuickSortAlgorithm;
         else if (stricmp(algorithmName, "tbbstableqsort")==0)
             sortAlgorithm = tbbStableQuickSortAlgorithm;
```

Comment:
Since a warning is logged below its good as is, I was initially thinking of something like:
```
\#else
OWARNLOG("TBB sorts are not in platform, falling back to default sort");
\#endif
type message.